src/main/kotlin/com/jetbrains/snakecharm/lang/parser/SnakemakeStatementParsing.kt

```diff
@@ -80,9 +114,17 @@ class SnakemakeStatementParsing(
         val scope = context.scope
 
         myBuilder.setDebugMode(false)
+        if (myBuilder.tokenType == PyTokenTypes.IDENTIFIER) {
+            // XXX: workaround/fix for #130:
+            val actualToken = SnakemakeTokenTypes.stringToTokenType[myBuilder.tokenText]
+            if (actualToken != null) {
+                myBuilder.remapCurrentToken(actualToken)
+            }
+        }
         val tt = myBuilder.tokenType
 
-        if (tt !in SnakemakeTokenTypes.WORKFLOW_TOPLEVEL_DECORATORS || scope.inParamArgsList) {
+        val parsingFunction = keywordToParsingFunction[myBuilder.tokenText]
```

Comment:
No clear why do we need here to check `myBuilder.tokenText` again, cacl it's hash, seems `tt = myBuilder.tokenType` already defines how to parse next piece of code.